manifests/qpid.pp

```diff
@@ -18,12 +18,14 @@
     require   => Service['qpidd'],
     logoutput => true,
   } ~>
-  exec { 'bind katello entitlements queue to qpid exchange messages that deal with entitlements':
-    command   => "qpid-config --ssl-certificate ${client_cert} --ssl-key ${client_key} -b 'amqps://localhost:5671' bind event ${candlepin_event_queue} '*.*'",
-    onlyif    => "qpid-config --ssl-certificate ${client_cert} --ssl-key ${client_key} -b 'amqps://localhost:5671' queues ${candlepin_event_queue}",
-    path      => '/usr/bin',
-    require   => Service['qpidd'],
-    logoutput => true,
+  qpid::bind_event {'entitlement.created':
```

Comment:
Currently you named the parameter ```candlepin_event_queue``` so it'd look as follows:
```puppet
qpid::bind_event { 'entitlement.created':
  candlepin_event_queue => $candlepin_event_queue,
}
```

But I'd rename the parameter to just ```queue``` so the second line becomes ```queue => $candlepin_event_queue```